src/Protocol/Diaspora.php

```diff
@@ -2145,8 +2164,10 @@ private static function receiveRequestMakeFriend(array $importer, array $contact
 	 * @return bool Success
 	 * @throws \Exception
 	 */
-	private static function receiveContactRequest(array $importer, $data)
+	private static function receiveContactRequest(array $importer, $data): bool
 	{
+		// @TODO Need to find object type, roland@f.haeder.net
+		Logger::debug('data='.get_class($data));
```

Comment:
Do you still need this? `$data` probably is a `stdClass` object.

Reply:
If you are sure?

Reply:
I have closed the other mentions as they are about the same added logger message. And I haven't found it yet.

Reply:
I have found `SimpleXMLElement` as a possible type. `receiveAccountDeletion()` receives `$fields` from `dispatch()` method which has that type-hint set.